src/inspector_agent.cc

```diff
@@ -837,8 +840,7 @@ void Agent::DisableAsyncHook() {
   }
 }
 
-void Agent::ToggleAsyncHook(Isolate* isolate,
-                            const Global<Function>& fn) {
+void Agent::ToggleAsyncHook(Isolate* isolate, const Local<Function>& fn) {
```

Comment:
```suggestion
void Agent::ToggleAsyncHook(Isolate* isolate, Local<Function> fn) {
```

for consistency with everything else?